Dockerfile

```
@@ -0,0 +1,5 @@
+FROM python:3.6
+
+ADD requirements.txt /safaribooks/requirements.txt
+WORKDIR /safaribooks
+RUN pip3 install -r requirements.txt
```

You could make the assumption the entrypoint would be safaribooks.py. Then the command to run this can be shortened to `docker-compose run safaribooks --cred "account_mail@mail.com:password01" XXXXXXXXXXXXX`

It also means this dockerfile can work standalone without the need for docker-compose or to mount the project files.

```suggestion
RUN pip3 install -r requirements.txt

COPY . /app

ENTRYPOINT [ "python3", "safaribooks.py"]
```

---

Hey @PaulDMooney,
Thank you for the suggestion. I  see your point and added `entrypoint` into `docker-compose.yml`
I would rather keep that part in docker-compose.yml otherwise each time when the code change the user has to run `docker-compose build`.